src/GitElephant/Command/Caller/Caller.php

```diff
@@ -81,21 +80,25 @@ public function __construct($gitPath, $repositoryPath)
      * @throws \Symfony\Component\Process\Exception\LogicException
      * @return Caller
      */
-    public function execute($cmd, $git = true, $cwd = null, $acceptedExitCodes = array(0)): \GitElephant\Command\Caller\CallerInterface
+    public function execute(string $cmd, bool $git = true, string $cwd = null, array $acceptedExitCodes = array(0)): CallerInterface
     {
         if ($git) {
             $cmd = $this->getBinaryPath() . ' ' . $cmd;
         }
-
         if (stripos(PHP_OS, 'WIN') !== 0) {
             // We rely on the C locale in all output we parse.
             $cmd = 'LC_ALL=C ' . $cmd;
         }
-
         if (is_null($cwd) || !is_dir($cwd)) {
             $cwd = $this->repositoryPath;
         }
-        $process = Process::fromShellCommandline($cmd, $cwd);
+
+        if (method_exists(Process::class, 'fromShellCommandline')) {
```

Comment:
Yea, seems best, yet simple solution here.. It's a shame symfony haven't introduced Process class interface, that should have been changed on updated api. Anyway, not a disaster.. )